assets/js/main.js

```diff
@@ -118,6 +118,20 @@ window.openRegister = function() {
 
 /*--------------------------------------------*/
 
+const onUsernameChecked = function(username, field) {
+    if (this.status !== 200) {
+        return;
+    }
+    const jsonObj = JSON.parse(this.responseText);
+    const message = jsonObj.message;
+    const exists = message.exists;
+    field.classList.add(`input-field-${exists ? "error" : "success"}`);
+    const label = field.nextElementSibling;
+    if (label) {
+        label.innerText = exists ? `${username} is not available` : "";
```

Comment:
From UX perspective I think I might actually want to display “username is available” if I’m not adding in checkmark.

Reply:
Done 
![Screenshot 2021-06-19 at 23-18-18 simpleimage - Simple Image Upload](https://user-images.githubusercontent.com/3276350/122661181-893f8980-d155-11eb-930e-1dcf74bcca86.png)